### DISCUSSION TITLE
Why we can not use Dual clip loader with SDXL without having a black image ?

### DISCUSSION THREAD

**Discussion by Creative-comfyUI** [comfyanonymous/ComfyUI#5574] [Q&A]
I am trying different combinations with no result. Does it work with all or some models? 
What is the solution?

Same problem with SD3.5 turbo GGUF black image ? 

It looks like this is a ComfyUI problem, as we have seen before.

**Comment by ltdrdata** [comment]
![workflow (35)](https://github.com/user-attachments/assets/a9a58560-a4a5-49a4-91af-42f4797fb6fd)

There is no problem.
What is your workflow?

**Comment by Creative-comfyUI** [comment]
It is a mess with all the clips we can use :-) . That's why I discover some works and others don't. I also discover how the clip has a great influence on the result and  enhance SDXL image and compostion 

More over about SD3.5, even if there is a randomised seed, we have the same images, I was surprised.

**Reply by ltdrdata** [reply]
Each technology uses clip models with their own recipes to improve performance.
It would be helpful if appropriate combination table information were provided in the hint section of the clip loader.

**Comment by Creative-comfyUI** [comment]
Where we found the hint section of the clip loader ?

**Reply by ltdrdata** [reply]
I submitted a PR.
https://github.com/comfyanonymous/ComfyUI/pull/5576
